src/components/PageHeader/PageHeader.module.css

```diff
@@ -71,6 +71,7 @@
  * 1) Kicker is an optional element that appears above the header
  */
 .page-header__kicker {
+  @mixin eds-typography-preset-009;
```

Comment:
We don't want to use tier 1 presets in components. Let's not bake in a page header kicker and let the `Text` component do the heavy lifting with the styling. This should just be a slot with maybe default spacing underneath